src/EnergyPlus/api/state.cc

```diff
@@ -59,6 +59,13 @@ EnergyPlusState stateNew()
     return reinterpret_cast<EnergyPlusState>(state);
 }
 
+EnergyPlusState stateNewPython()
+{
+    auto *state = new EnergyPlus::EnergyPlusData;
+    state->dataPluginManager->eplusRunningViaPythonAPI = true;
+    return reinterpret_cast<EnergyPlusState>(state);
+}
+
```

Comment:
When running from Python, set the boolean to true.

Reply:
Nice.